packages/edit-post/src/components/secondary-sidebar/list-view-sidebar.js

```diff
@@ -43,10 +57,49 @@ export default function ListViewSidebar() {
 			onKeyDown={ closeOnEscape }
 		>
 			<div
-				className="edit-post-editor__list-view-panel-header"
+				className="edit-post-editor__list-view-panel-header components-panel__header edit-post-sidebar__panel-tabs"
 				ref={ headerFocusReturnRef }
 			>
-				<strong id={ labelId }>{ __( 'List View' ) }</strong>
```

Comment:
This needs to stay for accessibility.

Reply:
I added an aria-label instead aria-labelledby it should have the same effect.